sw/device/silicon_creator/rom/rom_start.S

```diff
@@ -136,16 +145,22 @@ _rom_start_boot:
   la gp, __global_pointer$
   .option pop
 
+LABEL_FOR_TEST(kRomStartBootMaybeHalt)
   // Check if we should halt here.
   li   a0, (TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + \
             OTP_CTRL_SW_CFG_WINDOW_REG_OFFSET)
   lw   t0, OTP_CTRL_PARAM_CREATOR_SW_CFG_ROM_EXEC_EN_OFFSET(a0)
-  bnez t0, .L_exec_en
+  bnez t0, kRomStartBootExecEn
 .L_halt_loop:
   wfi
   j .L_halt_loop
 
-.L_exec_en:
+LABEL_FOR_TEST(kRomStartBootExecEn)
```

Comment:
nit: Can you use `.L_exec_en` here? If you cannot use it in tests, we can add `LABEL_FOR_TEST(kRomStartBootExecEn)` -- I hope it doesn't cause build issues.

I'd prefer defining as many labels for tests as needed but not having any references to them in `rom_start.S`.

Reply:
Sure, that makes sense. Just double checked that we're not using any of the for-test labels in `rom_start.S`.